### PR TITLE
Registration

### PR DESCRIPTION
**Pull request by AliciaRabbit**
Registration pages flow throughout the steps
Css styling almost on point

### CODE REVIEW

File: src/components/headerNav.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+import FlatButton from 'material-ui/FlatButton';
+import Toggle from 'material-ui/Toggle';
+import MoreVertIcon from 'material-ui/svg-icons/navigation/more-vert';
+import NavigationClose from 'material-ui/svg-icons/navigation/close';
+
+import Avatar from 'material-ui/Avatar';
+import RaisedButton from 'material-ui/RaisedButton';
+import '../css/index.css'
+
+class headerNav extends Component{
```
Comment (wvictor-rr) [inline src/components/headerNav.js]: Inconsistent with naming convention used for other classes.

File: .npmignore.txt [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# See https://help.github.com/ignore-files/ for more about ignoring files.
```
Comment (KimberleighBeyl) [inline .npmignore.txt]: what is this? Delete it or fix it

File: .vscode/launch.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```
Comment (KimberleighBeyl) [inline .vscode/launch.json]: this should be in the .gitignore

File: readme.txt [hunk 4]
```diff
@@ -0,0 +1,3 @@
+Yay this works
```
Comment (KimberleighBeyl) [inline readme.txt]: what is this.

File: src/components/headerNav.js [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+  state = {
```
Comment (KimberleighBeyl) [inline src/components/headerNav.js]: wrong

File: src/reducers/index.js [hunk 6]
```diff
@@ -0,0 +1,10 @@
+import { combineReducers } from 'redux'
+import { routerReducer } from 'react-router-redux'
+// TODO : Import all the reducers
+import reducers1 from './reducer1'
```
Comment (KimberleighBeyl) [inline src/reducers/index.js]: what is reducer1

File: .vscode/launch.json (hunk 3 above)
Comment (AliciaRabbit) [reply .vscode/launch.json]: deleted

File: src/components/headerNav.js (hunk 1 above)
Comment (AliciaRabbit) [reply src/components/headerNav.js]: This has been fixed

File: src/components/headerNav.js (hunk 5 above)
Comment (AliciaRabbit) [reply src/components/headerNav.js]: Fixed

File: src/components/headerNav.js (hunk 1 above)
Comment (AliciaRabbit) [reply src/components/headerNav.js]: This has been fixed please approve

**Pull request merged**